3a4
> import sys
21c22
< if not check_disk_usage("/", 2 * 2**30, 10):
---
> if not check_disk_usage("/", 2, 10):
23c24
<     return 1
---
>     sys.exit(1)
26c27
< return 0
---
> sys.exit(0)
--- health_check_original.py	2024-01-07 12:49:57.468523643 +0000
+++ health_check_fixed.py	2024-01-07 12:55:19.937039914 +0000
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
 import shutil
+import sys
 
 
 def check_disk_usage(disk, min_absolute, min_percent):
@@ -18,9 +19,9 @@
 # check for atleast 2 GB and 10% free
 
 
-if not check_disk_usage("/", 2 * 2**30, 10):
+if not check_disk_usage("/", 2, 10):
     print("ERROR : Not enough disk space")
-    return 1
+    sys.exit(1)
 
 print("Everything is OK !")
-return 0
+sys.exit(0)
